app/views/clients/hud_assessments/show.haml

```diff
@@ -1,4 +1,4 @@
-- title = 'HUD Assessment'
+- title = 'HUD Coordinated Entry Assessment'
```

Comment:
intake/update/exit are referred to as "HUD Assessments" so I think this is clearer